### PR TITLE
Add NEMO governing equations to theory.md

### PR DESCRIPTION
**Pull request by jtneedels**
Adds governing equations for NEMO solver (nonequilibrium NS and Euler) to theory.md. Work in conjunction with @WallyMaier.

### CODE REVIEW

**Review by pcarruscag — COMMENTED** [review]
What is the .DS_Store file?

File: _docs_v7/Theory.md [hunk 1]
```diff
@@ -87,6 +89,68 @@ Within the `EULER` solvers, we discretize the equations in space using a finite
 
 ---
 
+# Thermochemical Nonequilibrium Navier-Stokes #
+
+| Solver | Version | 
+| --- | --- |
+| `NEMO_NAVIER_STOKES`, `NEMO_RANS` | 7.0.0 |
```
Comment (pcarruscag) [inline _docs_v7/Theory.md:96]: Is NEMO_RANS operational already?

**Comment by jtneedels** [comment]
Oops, didn't see that got added. Must be some kind of cache file for when I viewed the md as an html on my machine, I'll remove it.

File: _docs_v7/Theory.md (hunk 1 above)
Comment (jtneedels) [reply _docs_v7/Theory.md:96]: Last I talked to @WallyMaier it works in his branch, but it's not in develop. I think he is still doing V&V. Should I remove the NEMO_RANS ref here until it's in develop?

**Review by pcarruscag — APPROVED** [review]

File: _docs_v7/Theory.md (hunk 1 above)
Comment (pcarruscag) [reply _docs_v7/Theory.md]: The documentation and code branches should be in sync (develop -> develop, master -> master)

**Pull request merged**